buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -103,6 +103,16 @@ object Versions {
   const val webkit: String = "1.3.0"
 
   const val junit: String = "1.1.4"
+
+  const val roomVersion = "2.3.0"
+
+
+  /**
+   * Current version: "6.2"
+   * See issue 19: How to update Gradle itself?
+   * https://github.com/jmfayard/buildSrcVersions/issues/19
+   */
+  const val gradleLatestVersion: String = "6.7.1"
```

Comment:
This code is unused here, we already removed this code in https://github.com/kiwix/kiwix-android/pull/3278/files . Please remove this code.

Reply:
From the rebase we get it! Done https://github.com/kiwix/kiwix-android/pull/3143/commits/30a04b8cb21f6abbdc22a040aa781affd6d6e5d2